test/CMakeLists.txt

```diff
@@ -256,6 +256,7 @@ ngen_add_test(
         NGen::forcing
     REQUIRES
         NGEN_WITH_NETCDF
+        NGEN_WITH_FORTRAN
```

Comment:
Did you mean `NGEN_WITH_BMI_FORTRAN`?

Though looking at these option names, the option should _probably_ be `NGEN_WITH_FORTRAN` instead (especially since we have `NGEN_WITH_PYTHON`).

Reply:
Oops, yeah, I did. Thanks for catching that. I wouldn't have noticed that the test was just *always* disabled, since we don't have any reporting of reduced code coverage.